docs/Apple/ws1intelligence.md

```diff
@@ -774,3 +774,39 @@ Swift
 ```Swift
 class func getUserUUID -> String
 ```
+
+
```

Comment:
Can the API `exportTelemetryFeatureData` also be added to this page please? The documentation and the API are in the release notes - https://github.com/euc-dev/ws1-sdk-intelligence/pull/8/files#diff-d45acb396ad072564aabce82f4a716f648f96ebe3419b6c30fb490ab4d2be882R80